geode-core/src/main/java/org/apache/geode/management/internal/api/LocatorClusterManagementService.java

```diff
@@ -157,20 +156,38 @@ public LocatorClusterManagementService(InternalConfigurationPersistenceService p
         validator.validate(CacheElementOperation.CREATE, config);
       }
 
-      // check if this config already exists on all/some members of this group
+      // check if this config already exists
       if (configurationManager instanceof CacheConfigurationManager) {
         memberValidator.validateCreate(config, (CacheConfigurationManager) configurationManager);
       }
-      // execute function on all members
     } catch (EntityExistsException e) {
       raise(StatusCode.ENTITY_EXISTS, e);
     } catch (IllegalArgumentException e) {
       raise(StatusCode.ILLEGAL_ARGUMENT, e);
     }
 
-    Set<DistributedMember> targetedMembers = memberValidator.findServers(group);
+    // find the targeted members
+    Set<String> groups = new HashSet<>();
+    Set<DistributedMember> targetedMembers;
+    if (config instanceof RegionAware) {
+      String regionName = ((RegionAware) config).getRegionName();
+      groups = memberValidator.findGroups(regionName);
+      if (groups.isEmpty()) {
+        raise(StatusCode.ENTITY_NOT_FOUND, "Region provided does not exist: " + regionName);
+      }
+      targetedMembers = memberValidator.findServers(groups.toArray(new String[0]));
```

Comment:
This varargs method forces too many other parts of the code to convert collections into arrays. Consider adding another version or two of `findServers()`—one that takes a single string, and another that takes an iterator or collection. Then our production code and test code won't have to jump through so many hoops to do something so simple.

Reply:
this makes passing in one single string or nothing easier. I updated the tests to not make the unnecessary conversions if passing in a single string or nothing.